konfera/models/talk.py

```diff
@@ -23,13 +24,20 @@
 
 class Talk(models.Model):
     title = models.CharField(max_length=256)
-    abstract = models.TextField()
+    abstract = models.TextField(help_text=_('Abstract will be published in the schedule.'))
     type = models.CharField(choices=TALK_TYPE, max_length=32)
     status = models.CharField(choices=TALK_STATUS, max_length=32)
-    duration = models.IntegerField(choices=TALK_DURATION)
+    duration = models.IntegerField(choices=TALK_DURATION, help_text=_('Talk duration in minutes.'))
     primary_speaker = models.ForeignKey('Speaker', related_name='secondary_speaker')
```

Comment:
(you didn't change this line but) I noticed that `primary_speaker` has related_name `secondary_speaker` an vice versa.